smarts/core/agent_manager.py

```diff
@@ -182,14 +182,14 @@ def observe(self, sim):
                     sim, agent_id, sensor_state, vehicle
                 )
 
-                if sim.vehicle_index.vehicle_is_hijacked(vehicle.id):
+                if sim.vehicle_index.vehicle_is_shadowed(vehicle.id):
```

Comment:
@JVillella I was wondering if you switched to using `is_hijacked` intentionally? According to the changes in PR #101 , `is_shadowed` was the original intention.

Reply:
The believe your changes are more appropriate, mine were contrived.